libs/sdk-ui-tests-e2e/Dockerfile

```diff
@@ -0,0 +1,28 @@
+# (C) 2021 GoodData Corporation
+
+FROM harbor.intgdc.com/tools/gdc-frontend-node:node-12.16.1-yarn-1.22.4 as build-stage
+
+ARG NPM_TOKEN
+
+WORKDIR /build
+
+COPY . .
+
+ENV FORCE_COLOR 0
+
+RUN npm config set '//registry.npmjs.org/:_authToken' $NPM_TOKEN \
+    && yarn install --frozen-lockfile \
+    && yarn dist
+
+FROM harbor.intgdc.com/3rdparty/nginxinc/nginx-unprivileged:1.17.2-alpine
+
+COPY --from=build-stage /build/nginx/nginx.conf /etc/nginx/nginx.conf
+COPY --from=build-stage /build/dist /usr/share/nginx/html/dashboards/
+#COPY --from=build-stage /build/app/assets/logo.png /usr/share/nginx/html/images/header/logo.png
```

Comment:
Forgotten commented out code, should we remove it?

Reply:
Same issue as the previous comment on Dockerfile.